apduWrapper.go

```diff
@@ -51,6 +52,8 @@ func ErrorMessage(errorCode uint16) string {
 		return "[APDU_CODE_INS_NOT_SUPPORTED] Instruction code not supported or invalid"
 	case 0x6E00:
 		return "[APDU_CODE_CLA_NOT_SUPPORTED] CLA not supported"
+	case 0x6E01:
+		return "[0x6E01] Ledger Connected but Chain Specific App Not Open"
```

Comment:
Can you add in the return the corresponding  apdu code instead of `0x6E01`?

Reply:
Could not locate the APDU description in any other source code/documentation.  Do you know what repo/component runs the ledger "home screen"?